api_etl/adapters/exampleIndividialAdapter.py

```diff
@@ -21,6 +21,9 @@ def transform(self, data: Any) -> Any:
                 raise self.Error("Invalid input, rows field required")
 
             for row in page["rows"]:
+                if "json_ext" in row and isinstance(row["json_ext"], dict):
```

Comment:
Do you think we can make it like:
```json_ext = row.get("json_ext")
    if isinstance(json_ext, dict):
        json_ext.pop("id", None)```
        
        it will be more readable, we can then avoid explict check